acceptance/setup/common/099_Archive_Logs.rb

```diff
@@ -0,0 +1,9 @@
+step 'Archive files created during tests' do
+  ## Copy file(s) from master
+  archive_file_from(master, '/var/log/pcp-broker.log')
+  
+  ## Copy file(s) from agents
+  agents.each do |agent|
+    archive_file_from(agent, '/var/log/puppetlabs/pxp-agent/pxp-agent.log')
```

Comment:
This won't work for Windows agents - there's a helper method to get the logfile for a given agent - https://github.com/puppetlabs/pxp-agent/blob/master/acceptance/lib/pxp-agent/test_helper.rb#L20